src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterOption } from '../../types/FilterOption';
+
+interface Props {
+  todos: Todo[];
+  filter: string;
+  setFilter: (filter: FilterOption) => void;
+}
+
+export const Footer: React.FC<Props> = ({ todos, filter, setFilter }) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">{`${todos.length} items left`}</span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filter === FilterOption.ALL,
+          })}
+          onClick={() => setFilter(FilterOption.ALL)}
+        >
+          All
```

Comment:
Can't you use the enum here as well? 
```suggestion
          onClick={() => setFilter(FilterOption.ALL)}
        >
         {FilterOption.ALL}
```